package.json

```diff
@@ -3,7 +3,16 @@
   "version": "0.0.0",
```

Comment:
Should change the `name` to something like `@ubiquity-dao/ubiquibot-kernel` to have it under our organization.